src/quo2/components/notifications/count_down_circle.cljs

```diff
@@ -0,0 +1,24 @@
+(ns quo2.components.notifications.count-down-circle
+  (:require
+   ["react-native-countdown-circle-timer" :refer [CountdownCircleTimer]]
+   [quo2.foundations.colors :as colors]
+   [quo2.theme :as theme]
+   [reagent.core :as reagent]))
+
+(def count-down-circle-timer (reagent/adapt-react-class CountdownCircleTimer))
```

Comment:
Hi @yqrashawn, Thank you for the great work, animations are looking good.

Can we move this to `quo.react-native`?

Reply:
I thought we shouldn't add things to quo anymore, and this also makes this component standalone.

Reply:
Not relevant anymore, if we are going to implement it in our code base https://github.com/status-im/status-mobile/pull/14376?notification_referrer_id=NT_kwDOAQTiGLM0ODQ3NTgwMjEwOjE3MDk3MjQw#discussion_r1039563360